binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -220,6 +240,9 @@ public void dispose() {
     public void initialize() {
         logger.debug("Initializing hue bridge handler.");
 
+        getConfig().put(CommonPropertyKey.VENDOR.name, HueBindingConstants.VENDOR_PHILIPS);
+        getConfig().put(CommonPropertyKey.MODEL.name, THING_TYPE_BRIDGE.getId());
```

Comment:
Didn't we discuss that "bridge" is not a valid model name and thus this should better not be set?